src/main/java/module/transfer/test/OverviewPanel.java

```diff
@@ -151,7 +151,7 @@ public int compare(Wage o1, Wage o2) {
 				// + wage.getWage() + " for age " + ageAt);
 			} else {
 				System.out
-						.println("####- " + player.getName() + " wage not found for age " + ageAt);
+						.println("####- " + player.getFullName() + " wage not found for age " + ageAt);
```

Comment:
This should probably become a logging statement?

Reply:
done